common/utils.cpp

```diff
@@ -343,7 +347,9 @@ void DBusHandler::setDbusProperty(const DBusMapping& dBusMap,
             }
             method.append(dBusMap.interface.c_str(),
                           dBusMap.propertyName.c_str(), variant);
-            bus.call_noreply(method);
+            bus.call_noreply(
+                method, std::chrono::duration_cast<microsec>(sec(DBUS_TIMEOUT))
```

Comment:
Rather than casting DBUS_TIMEOUT every time, can we set it to one common variable and use it?

Reply:
wil check if we can add it in utils.hpp

Reply:
this will be addressed in subsequent commits since the suggested change also has to be made in files that are not part of this commit